.github/workflows/build-ci-atlas.yml

```diff
@@ -1,94 +1,98 @@
 name: "Atlas CI"
 
 on:
-    push:
-    pull_request:
+  push:
+    branches:
+      - "[0-9]+.[0-9x]+"
+  pull_request:
+    branches:
+      - "[0-9]+.[0-9x]+"
```

Comment:
If we create a `feature/*` branch, the CI will not run on the PR targeting this feature branch.

Reply:
Added a commit to add this to all workflows.